engine/src/evm/rpc/node_interface.rs

```diff
@@ -0,0 +1,54 @@
+use ethers::prelude::*;
+use sp_core::H160;
+use std::str::FromStr;
+
+use anyhow::{Ok, Result};
+
+use super::{EvmRpcClient, EvmRpcSigningClient};
+
+abigen!(NodeInterface, "$CF_ARB_CONTRACT_ABI_ROOT/INodeInterface.json");
+
+// This is a kind of precompile on Arbitrum (although not deployed on chain)
+const NODE_INTERFACE_ADDRESS: &str = "0x00000000000000000000000000000000000000C8";
```

Comment:
What do you mean? Is there some kind of reference we can provide a link to here, or somewhere we can check this?

Reply:
https://docs.arbitrum.io/build-decentralized-apps/nodeinterface/reference
It's an implementation at a node level as if it were a contract deployed at that address but it's not really deployed. I'll improve that comment with the link.